orchestrator.py

```diff
@@ -37,14 +37,19 @@ def storing_and_sorting_items_data(db_queries, items_to_store):
     db_queries.add_prices(items_to_store)
     db_queries.check_for_lowest_price_and_update()
     id_list_to_pass = [(item[0],) for item in items_to_store]
```

Comment:
why are you getting it as a tuple? `id_list_to_pass` will become a list of tuples instead of a list of strings